test/kms_signing_test.go

```
@@ -0,0 +1,101 @@
+
+	client := infisical.NewInfisicalClient(context.Background(), infisical.Config{
+		SiteUrl: "http://localhost:8080",
+	})
+	_, err := client.Auth().UniversalAuthLogin("<>", "<>")
+
+	if err != nil {
+		panic(err)
+	}
+
+	randomBytes := make([]byte, 16)
+	rand.Read(randomBytes)
```

golangci-lint (1.64.8)

Error return value of `rand.Read` is not checked

(errcheck)